### PR TITLE
Update regression tests to use new content tree (Issue 2418 task 6)

### PR DESCRIPTION
**Pull request by alflennik**
See https://github.com/w3c/aria-practices/issues/2418 for more information about the task.

### CODE REVIEW

**Review by mcking65 — CHANGES_REQUESTED** [review]
Thank you @alflennik! This is great!! Found only a few minor issues and a few questions.

File: test/tests/breadcrumb_index.js
Comment (mcking65) [inline test/tests/breadcrumb_index.js:5]: For consistency, should we also change the name of the test js file? "test/tests/breadcrumb_index.js" might lead someone to think this is for a different example, maybe?

File: content/patterns/treeview/examples/treeview-1/treeview-1a.html
Comment (alflennik) [reply content/patterns/treeview/examples/treeview-1/treeview-1a.html:262]: Nice catch Matt!

File: content/shared/js/app.js
Comment (alflennik) [reply content/shared/js/app.js:76]: This change is dependent on https://github.com/w3c/aria-practices/pull/2402 being merged. Once it's merged I'll rebase this branch and make the recommended change.

File: test/tests/breadcrumb_index.js
Comment (alflennik) [reply test/tests/breadcrumb_index.js:5]: Great catch, I didn't think of that. Updated.

File: cspell.json
Comment (alflennik) [reply cspell.json]: Sounds good! Removed.

File: content/shared/js/app.js
Comment (alflennik) [reply content/shared/js/app.js:70]: Same as below - this change is dependent on https://github.com/w3c/aria-practices/pull/2402 being merged. Once it's merged I'll rebase this branch and make the recommended change.

**Review by mcking65 — APPROVED** [review]

File: .github/workflows/spelling.yml
Comment (howard-e) [inline .github/workflows/spelling.yml:12]: Maybe this is covered in another PR/task and doesn't block this PR, but seems this should also be updated to `"!content/patterns/landmarks/examples/css/bootstrap.css"`.

File: .github/workflows/spelling.yml
Comment (howard-e) [inline .github/workflows/spelling.yml:22]: Same comment as above.

**Review by howard-e — APPROVED** [review]
LGTM

**Pull request merged**
